src/Tribe/Linked_Posts.php

```diff
@@ -959,6 +940,85 @@ public function render_meta_box_sections( $event ) {
 	}
 
 	/**
+	 * A several-step process that prints the "Create or Find {Linked Post Type Name}" labels.
+	 *
+	 * Numerous steps and caveats are covered in this method so that we can make these labels, which
+	 * are rather important, as translation-friendly as possible.
+	 *
+	 * @since TBD
+	 *
+	 * @param object $post_type The linked post type whose label is being rendered.
+	 * @param boolean $creation_enabled Whether the current user can create post types. If false, they can only add existing ones.
+	 *
+	 * @return string
+	 */
+	public function get_create_or_find_labels( $post_type, $creation_enabled  ) {
+
+		$plural_name             = $this->linked_post_types[ $post_type ]['name'];
+		$singular_name           = ! empty( $this->linked_post_types[ $post_type ]['singular_name'] ) ? $this->linked_post_types[ $post_type ]['singular_name'] : $plural_name;
+		$singular_name_lowercase = ! empty( $this->linked_post_types[ $post_type ]['singular_name_lowercase'] ) ? $this->linked_post_types[ $post_type ]['singular_name_lowercase'] : $singular_name;
+
+		// First, determine what indefinite article we should use for the post type (important for English and several other languages).
+		$indefinite_article = _x( 'a', 'Indefinite article for the phrase "Find a {post type name}. Will be replaced with "an" if the {post type name} starts with a vowel.', 'the-events-calendar' );
+
+		$post_type_starts_with = substr( $singular_name, 0, 1 );
+		$post_type_starts_with = strtolower( $post_type_starts_with );
+		$english_vowels        = array( 'a', 'e', 'i', 'o', 'u' );
```

Comment:
I've no idea as to how many other languages have a concept of indefinite articles ... however it seems like if `$indefinite_article` is translatable then the range of vowels we test against should also be translatable or at the very least filterable.

Fictional language Borkwebish - where the indefinite article is generally _zog_ unless the following word starts with _p, q_ or _r_ (in which case it is _zag)_ - provides a good example of the sort of problem I mean.

Reply:
... on second look, I see this logic isn't new, so don't worry

Reply:
cool. will leave for now